### ISSUE TITLE
"You provided the name of a praw.ini configuration which does not exist."

### ISSUE THREAD

**Issue by TimelordJace** [rhiever/reddit-twitter-bot#12]
I cloned your project into Visual Studio, and when I run the code, after a few seconds [this](https://gyazo.com/97fc9e9d9dd9e10bfdfe9794bdb28893) error appears. Do you have any clue as to how I could fix this error?

**Comment by cordeous** [comment]
i'm getting the same problem here

**Comment by driftshift** [comment]
Also having this problem, not sure how to proceed from here.

**Comment by myelmer** [comment]
You should check for praw documentations see [here.](https://praw.readthedocs.io/en/latest/getting_started/quick_start.html). I think there is a missing code piece in here. See [sentdex](https://www.youtube.com/watch?v=NRgfgtzIhBQ). He explains how to use it then you will be able to use it.

**Comment by AddaxSoft** [comment]
tl;dr: this solved my issue:

`pip install praw==3.6.0`
